sshconfig

```diff
@@ -4,3 +4,4 @@ Host i-*
     StrictHostKeyChecking no
     UserKnownHostsFile /dev/null
     LogLevel ERROR
+    ServerAliveInterval 60
```

Comment:
Running locally, I needed a "PatrickAliveInterval" after about 7 minutes of no output :-/

Reply:
papermill does not send output to stdout. remember that this is intended to be run in CI.

Reply:
It gets run in CI and by people who are testing notebooks to provide to us.  I'm thinking more about the latter.  I'm planning to add a couple of echo statements that will at least show something happening and indicate success.